packages/components/addon/components/hds/button-set/index.hbs

```diff
@@ -0,0 +1,4 @@
+{{! TODO: Add optional tag variations? p tag, etc. vs just div? }}
```

Comment:
Is just a div ok? Ideally I would want to add options to use various tags but let me know. (I will delete this comment before merging.)

Reply:
At the moment we don't provide this functionality in our component (having dynamic tags in Ember is not possible, see for example what needs to be done to achieve something similar in this PR: https://github.com/hashicorp/cloud-ui/pull/2574/files#diff-aca16c63293e60d44e3b43fa1db3a55266ac4811697d1c4d20dcf35b81c62f70)

Reply:
A div is fine, we shouldn't (and don't) provide options to change the tag name. 

I think this could be a template-only component. I don't think it needs a backing JS file; since there's just one class, the class name can go into the template. I also don't think we need `...attributes` but I am open to hearing some thoughts on what some use cases might be.

Reply:
Would end users be able to add additional attributes if needed if I removed `...attributes`? For example if they wanted to add an aria-label or data attribute or something for some purpose?

Reply:
Agree with @KristinLBradley here: we tend to add `...attributes` to all the components so consumers can easily add attributes to them. Most (all?) of our components work this way.